components/landing/topCollections/TopCollections.vue

```diff
@@ -1,8 +1,10 @@
 <template>
   <div>
-    <div class="flex justify-between mobile">
-      <div class="title is-2">{{ $t('general.topCollectionsHeading') }}</div>
-      <div class="top-collection-controls buttons items-start pt-2">
+    <div class="flex justify-between mobile !pb-9 md:!pb-10">
+      <div class="title is-2 md:!mb-0">
+        {{ $t('general.topCollectionsHeading') }}
+      </div>
+      <div class="top-collection-controls buttons items-start pt-2 !mb-0">
```

Comment:
why `!important`?

Reply:
adding any padding/margin class ends up using the bulma class and not the tailwindcss one , not needed here since bulma only goes up to -6 , I was changing the amount and forgot remove it